language/stdlib/modules/ValidatorConfig.move

```diff
@@ -145,11 +144,14 @@ module ValidatorConfig {
     }
 
     spec fun remove_operator {
-        include Roles::AbortsIfNotValidator;
+        include Roles::AbortsIfNotValidator; // Aborts if not validator [B24].
         let sender = Signer::spec_address_of(account);
         include AbortsIfNoValidatorConfig{addr: sender};
         ensures !spec_has_operator(Signer::spec_address_of(account));
         ensures spec_get_operator(sender) == sender;
+        ensures forall addr: address where addr != sender:
+            global<ValidatorConfig>(addr).operator_account == old(global<ValidatorConfig>(addr).operator_account); // The sender can only change its own operator account [B24].
```

Comment:
And here.

Ok I'm stopping now commenting on this issue. I would recommend to run something like `grep '\[B.*\]' *.move` to ensure all references to requirements are in proper doc comments.

Reply:
Done.